code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/state/SmallImageCardUIState.kt

```diff
@@ -12,12 +12,18 @@
 package com.adobe.marketing.mobile.aepcomposeui.aepui.state
 
 /**
- * Data class representing the UI state of smallImage template card.
+ * Class representing the UI state of a Small Image template card.
  *
  * @property dismissed Indicates whether the card has been dismissed.
  * @property selected Indicates whether the card is selected.
+ * @property read Indicates whether the card has been read.
+ * @property timerDisplayed Indicates whether the timer has been displayed.
+ * @property cardExpanded Indicates whether the card has been expanded.
  */
 data class SmallImageCardUIState(
-    override val dismissed: Boolean = false,
-    override val selected: Boolean = false
-) : AepCardUIState
+    var timerDisplayed: Boolean = false,
```

Comment:
Can we add it when we plan to implement this state usage? It be clearer to see the implementation of the state.

Reply:
sure, i'll remove thse for now